### ISSUE TITLE
Cannot alias Package[pecl-memcached] to ["memcached"]

### ISSUE THREAD

**Issue by pkordylewski** [mayflower/puppet-php#74]
Howdy,

i'm getting the following error:

```
Evaluation Error: Error while evaluating a Resource Statement, 
Cannot alias Package[pecl-memcached] to ["memcached"] at xxx/modules/php/manifests/extension.pp:90; 
resource ["Package", "memcached"] already declared at xxx
```

I have a Memcached-Server installed (same package name). Can i somehow prevent this naming convention conflict?

Thanks,
Patryk

**Comment by fpletz** [comment]
Hi, thanks for your report! This is naming conflict is indeed a problem with the module and should be fixed on our side. Unfortunately you can't fix this in your code right now.

I will look at this on friday. We will release a fix with 3.2.0, probably also on friday.

**Comment by pkordylewski** [comment]
Ok, thanks!

**Comment by pkordylewski** [comment]
Any ETA on this? :-)

**Comment by fpletz** [comment]
@pkordylewski We've just released 3.2.0. Sorry for the long wait. :beers:

**Comment by pkordylewski** [comment]
Thanks, now it works :+1:
